pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -75,21 +89,29 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     interface IDeveloperIdProvider
     {
```

Comment:
Do we want to add the new methods to the existing interface as technically that would be breaking?  Instead, typically you would create a new `IDeveloperIdProvider2` interface or something similar with the new contract version and add them there.  The new interface can require this interface to make it such that you only need to implement one.

Reply:
The plan is to not update the contract version and take the breaking changes. Therefore, we can keep the new methods in the existing interface. Thanks!